Update model-query-history.md (#6990)
dbt-labs/docs.getdbt.com: website/docs/docs/collaborate/model-query-history.md

<Lightbox src="/img/docs/collaborate/dbt-explorer/model-query-credentials.jpg" width="50%" title="Confirm your deployment credentials in your environment settings page." />

4. Copy or cross reference those credential permissions with the warehouse permissions and grant your user the right permissions.
   

>    - #### For Snowflake

Reviewer: If we want to bullet point this and make it stand out, I think we should make the text bold rather than a header. If it's going to be a header (even an H4) I think we should remove the bullet
```suggestion
#### Snowflake model query history
```

>      This feature makes use of metadata tables available to Snowflake Enterprise tier accounts or higher, `QUERY_HISTORY` and `ACCESS_HISTORY`. The Snowflake user used in the production environment must have `GOVERNANCE_VIEWER` permissions to view the data.

Reviewer: ```suggestion
     Model query history makes use of metadata tables available to [Snowflake Enterprise tier](https://docs.snowflake.com/en/user-guide/intro-editions#enterprise-edition) accounts or higher, `QUERY_HISTORY` and `ACCESS_HISTORY`. The Snowflake user in the production environment must have the `GOVERNANCE_VIEWER` permission to view the data.
```

>      Before enabling Model Query History, your `ACCOUNTADMIN` must run the following grant statement in Snowflake to ensure for access:

Author: ```suggestion
     Before enabling Model query history, your `ACCOUNTADMIN` must run the following grant statement in Snowflake to ensure for access:
```

>      Before enabling Model query history, your `ACCOUNTADMIN` must run the following grant statement in Snowflake to ensure for access:

Reviewer: Suggest slight alteration because I believe other roles can run this statement
```suggestion
     Before enabling model query history, your account admin must run the following grant statement in Snowflake to ensure for access:
```

Author: ok thanks @matthewshaver ! going to cc @jzhu13 @matt-winkler about whether snowflake's `ACCOUNTADMIN` should can assign  `GOVERANCE_VIEWER` or if other roles can assign it?

Author: hey @matthewshaver , @jzhu13 has confirmed that [snowflake docs](https://docs.snowflake.com/en/sql-reference/account-usage#enabling-other-roles-to-use-schemas-in-the-snowflake-database) only mentions `access to schemas in this database can be granted by a user with the ACCOUNTADMIN role` and since it's explicitly mentioned in the snowflake doc -- we want to align with that.

     ```sql
     GRANT DATABASE ROLE SNOWFLAKE.GOVERNANCE_VIEWER TO ROLE <YOUR_DBT_CLOUD_DEPLOYMENT_ROLE>;
     ```

>      Without this grant, Model query history won't display any data. For more details, view the snowflake docs [here](https://docs.snowflake.com/en/sql-reference/account-usage#enabling-other-roles-to-use-schemas-in-the-snowflake-database). 

Reviewer: If we're treating it like its a proper noun, we should state "Model Query History," otherwise sentence case the whole thing
```suggestion
     Without this grant, model query history won't display any data. For more details, view the snowflake docs [here](https://docs.snowflake.com/en/sql-reference/account-usage#enabling-other-roles-to-use-schemas-in-the-snowflake-database). 
```

>     - ##### For BigQuery

Reviewer: Same feedback as above
```suggestion
##### BigQuery model query history
```

>       This feature uses the metadata from the `INFORMATION_SCHEMA.JOBS` view in BigQuery. To access this, the user configured for your production environment must have the following [IAM roles](https://cloud.google.com/bigquery/docs/access-control) for your BigQuery project:

Reviewer: ```suggestion
Model query history uses the metadata from the `INFORMATION_SCHEMA.JOBS` view in BigQuery. To access this, the user configured for your production environment must have the following [IAM roles](https://cloud.google.com/bigquery/docs/access-control) for your BigQuery project:
```

Author: editing this to add a space btw history and uses @matthewshaver !
